common/src/main/java/tds/common/web/advice/ExceptionAdvice.java

```diff
@@ -36,6 +38,8 @@
     @ResponseStatus(value = HttpStatus.BAD_REQUEST)
     @ResponseBody
     ResponseEntity<ExceptionMessageResource> handleArgumentMismatchException(final MethodArgumentTypeMismatchException ex) {
+        LOG.info("Request made with invalid arguments ", ex);
```

Comment:
I thought `info` was okay because we're just recording that it happened and shipping it back to the caller, who can decide for themselves what the severity is.  I could see maybe elevating these to `warn`.